LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2017 Petr Brzek
+Copyright (c) 2018 Petr Brzek
```

Comment:
Legally this needs to remain 2017 or be adjusted to 2017-2018, otherwise it forfeits the copyright of code provided prior to 2018. See https://stackoverflow.com/questions/2390230/do-copyright-dates-need-to-be-updated for more information